module_utils/network_lsr/argument_validator.py

```diff
@@ -1553,6 +1553,9 @@ def get_default_bond(self):
 
 
 class ArgValidator_DictInfiniband(ArgValidatorDict):
+    # The default `p_key` value is -1, which means that the `p_key` is unset
```

Comment:
IMHO the comment is not necessary since it basically describes the meaning of that is deductible from the constant's name.

Reply:
The comment dropped. Thanks.